src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -118,11 +154,27 @@
                                       (reset! ready-to-lock? false))
                                     (and (not @locked?) (not @reviewing-audio?))
                                     (do
+                                      (audio/stop-recording
+                                       @recorder-ref
+                                       (cond
+                                         (and @ready-to-send? on-send)
+                                         #(do
+                                            (on-send (audio/get-recorder-file-path @recorder-ref))
+                                            (reload-recorder-fn))
+                                         (and @ready-to-delete? on-cancel)
+                                         #(do
+                                            (reload-recorder-fn)
+                                            (on-cancel))
+                                         :else
+                                         #(reload-recorder-fn))
+                                       #(log/error "[record-audio]" %))
                                       (reset! recording? false)
                                       (reset! ready-to-send? false)
```

Comment:
not sure about this, any chance there's useful info for debug in `@recorder-ref`

Reply:
Nope but just improved the log wordiness and added a log/debug for the success case, wasn't supposed to be like this